src/features/Dashboard/components/RecentTransactionList.tsx

```diff
@@ -126,12 +131,35 @@ export default function RecentTransactionList({ accountsList, className = '' }:
           address: recipient,
           networkId
         };
+        if (valueTransfers.length == 0) {
+          valueTransfers.push({
```

Comment:
Why is this done here? Can we do this functionally and also please test.

Reply:
The component shouldn't need to add things to this?

Reply:
We don't want the actual txReceipt to have a zero value transfer in it's arr of value transfers, but we DO want to display 0 ETH on a transaction in the RecentTransactionsList if the tx has no value transfers to improve UX of the Recent Txs panel. Left a comment to explain that.